canvas/image.go

```diff
@@ -91,16 +232,8 @@ func NewImageFromResource(res fyne.Resource) *Image {
 // The method for scaling can be set using the Fill field.
 func NewImageFromImage(img image.Image) *Image {
 	ret := &Image{}
-	ret.PixelColor = func(x, y, w, h int) color.Color {
-		size := img.Bounds().Size()
-		if size != image.ZP {
-			ret.PixelAspect = float32(size.X) / float32(size.Y)
-		}
-
-		hScale := float32(size.X) / float32(w)
-		vScale := float32(size.Y) / float32(h)
-
-		return img.At(int(float32(x)*hScale), int(float32(y)*vScale))
+	ret.Raster = func(w, h int) image.Image {
```

Comment:
This initially confused me - should a comment (or _, _ img) be used to indicate that we can ignore the size requested?

Reply:
No. It is still good to know what the arguments are, especially if we are not using them.